ext/tensorflow.mk

```diff
@@ -142,7 +142,7 @@ CUDA_MODELS=30 35 37 52 60 61
 
 # Which CUDA compute capabilities do we support?  3.0 is needed for the
```

Comment:
I'll revert that to 3.0 since the tests work fine with it and using something higher would break compat with AWS g2 instances.